src/nav/MainNavBar.js

```diff
@@ -18,6 +18,12 @@ import {
   getTitleTextColor,
 } from '../selectors';
 
+const componentStyles = StyleSheet.create({
+  hidden: {
```

Comment:
In React we tend not to hide but to not render. This is more performant and what we do everywhere else.